disnake/user.py

```diff
@@ -55,6 +59,7 @@ class BaseUser(_UserTag):
         "_accent_colour",
         "_public_flags",
         "_state",
+        "collectibles",
```

Comment:
Can't suggest, but this sould be moved right under `system` to be consistent

Reply:
https://github.com/DisnakeDev/disnake/pull/1303/commits/2f2cc13628b60041deb27ad68426f0273e5b65db